src/Resources/views/CRUD/display_enum.html.twig

```diff
@@ -10,13 +10,16 @@ file that was distributed with this source code.
 #}
 
 {%- apply spaceless %}
-    {% set value = use_value|default(false) ? value.value : value.name %}
-
-    {% if translation_domain|default(null) is null %}
-        {% set value = value %}
+    {% if value is null %}
```

Comment:
In order to cover this case and avoid further regressions, please add a case at [`RenderElementExtensionTest::getRenderListElementTests()`](https://github.com/bedfir/SonataAdminBundle/blob/cf2cc488d6f9836ac3c2cc8e98906142c8aed45b/tests/Twig/Extension/RenderElementExtensionTest.php#L1541):

```php
$elements[] = [
    '<td class="sonata-ba-list-field sonata-ba-list-field-enum" objectId="12345"> &nbsp; </td>',
    FieldDescriptionInterface::TYPE_ENUM,
    null,
    [
        'use_value' => true,
    ],
];
```